ProboCI/probo
# Feature/lliss/adjust wordpress behaviors

This updates and fixes functionality in the wordpress plugin.

Changes include

- Use constants throughout the codebase
- Ensure the wordpress database name can be properly set
- Ensure file uploads succeed
- Adjust the options allowed on the wordpress plugin.

### To Test
- Create a wordpress site and attempt to build it using probo.
- See https://github.com/lliss/wordpress-test/pull/8 for helpful demos.
